### PR TITLE
Add an example of how to use --vrrp-id-offset to configure multiple high available services in the same network.

### PR DESCRIPTION
**Pull request by ramr**
@adellape / @bfallonf / @ahardin-rh   PTAL  thx.

/cc @bmeng

### CODE REVIEW

**Comment by ramr** [comment]
@bfallonf @adellape   /bump

File: admin_guide/high_availability.adoc [hunk 1]
```diff
@@ -566,3 +566,57 @@ additional virtual IP addresses:
...
 <1> `10.245.2.110,10.245.2.201-205` have been added to the list.
 ====
+
+
+[[multiple-highly-available-services-in-a-network]]
+=== Multiple Highly Available Services In a Network
+
+The IPFailover service uses VRRP (Virtual Router Redundancy Protocol) to
+communicate with its peers. By default, the generated Keepalived
+configuration uses a vrrp id offset starting from 0 (and sequentially
```
Comment (bfallonf) [inline admin_guide/high_availability.adoc]: s/vrrp id/VRRP ID

File: admin_guide/high_availability.adoc [hunk 2]
```diff
@@ -566,3 +566,57 @@ additional virtual IP addresses:
...
+increasing) to denote the peers in a network.
+If you wish to run multiple highly available services in the same network
+(have multiple IP Failover deployments), you need to ensure that there is
+no overlap of the vrrp ids by using a different starting offset for your
```
Comment (bfallonf) [inline admin_guide/high_availability.adoc]: s/vrrp ids/VRRP IDs

**Comment by bfallonf** [comment]
@ramr A few typo-level comments, but it should be good otherwise.

**Comment by ramr** [comment]
@bfallonf made the requested changes - thx.

**Comment by bfallonf** [comment]
Looks good @ramr . Thanks! I'll merge.

**Comment by bfallonf** [comment]
Actually, @ramr to which version does this apply? Is this fixing something previously existing, or is this for 3.4?

**Comment by ramr** [comment]
@bfallonf  its an addendum to the doc to call out how to run multiple ipfailover configs  (keepaliveds) on the same network/cluster. Its a note/clarification to existing functionality, so ...

**Comment by bfallonf** [comment]
Thanks @ramr . I'll merge.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[High Availability]
|Added the xref:../admin_guide/high_availability.adoc#multiple-highly-available-services-in-a-network[Multiple Highly Available Services In a Network] section.
%

**Pull request merged**